i18n/locales/source/dashboard/scripts.yml

```diff
@@ -27038,11 +27172,17 @@ en:
             Responding to Words:
               name: Responding to Words
               description_student: Make the app respond to what the user chooses in exciting new ways.
-              description_teacher: In this lesson, students add conditionals to their projects, which will make their apps more responsive. Conditionals allow students to incorporate if-then logic to their code, which leads to more personalization in their app. Students can create programs that make new effects or sprites based on which word the user chooses.
+              description_teacher: |-
+                **Note:** This is an **optional** lesson in the module
+
+                In this lesson, students add conditionals to their projects, which will make their apps more responsive. Conditionals allow students to incorporate if-then logic to their code, which leads to more personalization in their app. Students can create programs that make new effects or sprites based on which word the user chooses.
             Poetry Circle:
               name: Poetry Circle
               description_student: 'Share your new poem apps with your friends! '
-              description_teacher: In this lesson, students will take turns sharing their poem app with their peers. Students start by discussing the importance of giving and getting feedback. Then, students conduct a poetry pair-share by sharing their poem apps with a partner and giving and getting feedback. Finally, the whole class will have a Poetry Circle followed by a discussion on the mood and tone, as well as visual effects, of a student volunteer’s poem.
+              description_teacher: |-
```

Comment:
TIL about the `|-` indicators in YAML
- `|` Literal style turns every newline within the string into a literal newline, and adds one at the end:
- Stripping is specified by the `-` chomping indicator. In this case, the final line break and any trailing empty lines are excluded from the scalar’s content.

Source: https://stackoverflow.com/a/21699210